src/Miew.js

```diff
@@ -2968,10 +2969,9 @@ Miew.prototype.setPivotAtom = function (atom) {
     return;
   }
 
-  const pos = this._gfx.pivot.position;
-  pos.copy(atom._position);
-  pos.applyMatrix4(visual.matrix);
-  pos.negate();
+  const center = atom._position.clone();
```

Comment:
One again, clone() means newly creation, isn't it?

Reply:
Yes, change it